client/my-sites/checkout/src/hooks/use-create-payment-methods/index.tsx

```diff
@@ -361,6 +410,10 @@ export default function useCreatePaymentMethods( {
 	stripeLoadingError: StripeLoadingError;
 	stripeConfiguration: StripeConfiguration | null;
 	stripe: Stripe | null;
+	isRazorpayLoading: boolean;
```

Comment:
Something to consider is if you want to [pause loading checkout](https://github.com/Automattic/wp-calypso/blob/59fd3281e67abdc8b126abe0f3b551647edad0c8/client/my-sites/checkout/src/components/checkout-main.tsx#L554) until Razorpay has loaded. I'm not sure if it's necessary or not to prevent any UX issues. Probably better if you don't need it because checkout load times are a highly sensitive topic but I wanted to mention something.

Reply:
I think it's not necessary to pause loading checkout for this; we can revisit that if it causes issues.